index.html

```diff
@@ -0,0 +1,74 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="css/style.css">
+    <title>mint.com signup page</title>
+</head>
+<body>
+    <!-- Logo -->
+    <header>
+        <div class="header">
+            <img src="images/logo.png" alt="Intuit Logo">
+        </div>
+    </header>
+    <!-- Main container-->
+    <main>
+        <!-- Form container -->
+        <div class="container">
+            <!-- Form heading -->
+            <div class="form-header text-center">
+                <h1>Create an Intuit account</h1>
+                <span class="sub-title">
+                    One account for everything Intuit, including Mint. <a href="https://accounts-help.lc.intuit.com/questions/1582580-creating-an-account">
+                        Learn more</a><br>
+                    Already have an account? <a href=#>Sign In</a>
+                </span>
+            </div>
+            <!-- Form -->
+            <form>
+                <fieldset class="form-main">
+                    <label for="email">Email address</label>
+                    <input type="email" name="email" id="email">
+
+                    <label for="phone">Phone <span class="font-300">(recommended)</span></label>
+                    <input type="tel" name="phone" id="phone">
+
+                    <label for="password">Password</label>
+                    <input type="password" name="password" id="password">
+
+                    <label for="confirm-password">Confirm Password</label>
+                    <input type="password" name="confirm-password" id="confirm-password">
+                </fieldset>
```

Comment:
- [ ] Kindly remove the confirm-password field as it is not on the source page.

Source | Clone
-- | --
![image](https://user-images.githubusercontent.com/57414789/83826536-c26b9f80-a6a1-11ea-8900-d0fbeeccb42c.png) | ![image](https://user-images.githubusercontent.com/57414789/83826555-cb5c7100-a6a1-11ea-9849-2d5db34e2838.png)